osmaxx-symbology/OSMaxx_point_symbols/svg/README.md

```diff
@@ -0,0 +1,19 @@
+## **svg to ttf automated conversion process**
+
+-	Clone pysvg2font module from [carrasti/pysvg2font](https://github.com/carrasti/pysvg2font.git)
+-	Generate the ttf file as instructed on the documentation of the repository
+-	You can recheck the result of the generated font using [fontforge](https://fontforge.github.io/en-US/).
+
+
+Throughout the process, error 
+	`Failed to parse colour` 
+will be generated. However, this error can be safely neglected.
+
+pysvg2font module was modified to result alphabetically ordered font.
+From `pysvg2font/pysvg2font/__init__.py` line 94:
+
+	`file_path.sort()`
+
+
+
+

```

Comment:
> I just renamed this file, it was already markdown.

Ah, @beneditatan converted it to Markdown in 0fc311104523d5b68ffd254b536ef84d4a3dbac9. Somehow, her commit became part of this pull request.